### ISSUE TITLE
[Bug]: Notifications Not Showing

### ISSUE THREAD

**Issue by JordanDalton** [NativePHP/laravel#48]
### What happened?

I'm unable to successfully create a notification. I'm using the documentation from Native's website pertaining to [Notifications](https://nativephp.com/docs/1/the-basics/notifications).

I've noticed that the path to the facade, `Native\Laravel\Facades\Notification`, is not in Native's file path.

Here's the shared trace for it: https://flareapp.io/share/95Jll42P

### How to reproduce the bug

Use the following snippet from: https://nativephp.com/docs/1/the-basics/notifications

`use Native\Laravel\Facades\Notification;
 
Notification::title('Hello from NativePHP')
    ->message('This is a detail message coming from your Laravel app.')
    ->show();`

### Package Version

0.1.0

### PHP Version

8.2.8

### Laravel Version

10.15.0

### Which operating systems does with happen with?

macOS

**Comment by khalmurad** [comment]
Hi.
I used like this:

```
$client = new Client();
$notification = new Notification($client);
$notification->title('Hello from NativeApp')
     ->message('This is a detailed message coming from your native app.')
     ->show();
```

The reason is that the Notification class is not in the place written in the documentation, but it is located in this place: `Native\Laravel\Notification`

In addition, the title function in the Notification class is not static.

**Comment by balduinofernando** [comment]
Thanks @khalmurad 

This solution you pointed worked out fine...

![Screenshot from 2023-07-22 16-42-30](https://github.com/NativePHP/laravel/assets/20538071/47c8bd7c-b1ff-47c2-8933-87afa6fc127c)

**Comment by JordanDalton** [comment]
The update removed the error, but a notification isn't appearing.

**Comment by mdloop** [comment]
The this time, the latest published package is not synced with the documentation.

```
use Native\Laravel\Client\Client;
use Native\Laravel\Notification;

$notification->title('Hello from NativePHP')
  ->message('This is a detail message coming from your Laravel app.')
  ->show();
```

Make sure you are invoking this from within the Electron (or eventually Tauri) since if you are using artisan commands or just visiting your site via the browser, there's nothing to receive the notification.

**Comment by multipacman** [comment]
You could also achieve this using only the Notification import since they have already initialised a client with the `new` function.

```php
use Native\Laravel\Notification;

 Notification::new()
            ->title('Hello from NativePHP')
            ->message('This is a detail message coming from your Laravel app.')
            ->show();
```

**Comment by ArielMejiaDev** [comment]
@JordanDalton do you check your settings, Apple requires to allow for every single app permission to show notifications.

**Comment by abishekrsrikaanth** [comment]
Is there a way to change the icon on the notifications? Do see any docs for that. I am assuming that changing the icon for the app will automatically do that, but still not sure how to do that.

**Comment by simonhamp** [comment]
The [Notification facade](https://github.com/NativePHP/laravel/blob/main/src/Facades/Notification.php) was added in #25 and was released in [v0.1.1](https://github.com/NativePHP/laravel/releases/tag/0.1.1)

@abishekrsrikaanth please could you raise the icon issue as a feature request?

**Comment by abishekrsrikaanth** [comment]
@simonhamp the feature request templates was not working, is that fixed?
